Makefile

```diff
@@ -148,6 +145,8 @@ push_images:
 	fi
 
 push_manifest:
+	# SECRET is used when pushing the manifest.
+	SECRET := $(shell echo "https://gcr.io" | docker-credential-gcr get | jq '.Secret')
```

Comment:
Comments left on the PR as a whole; changing the existing `:=` to `=` should fix our issue. (github made me add a comment here too...)